autopush-common/src/reliability.rs

```diff
@@ -76,8 +75,21 @@ impl PushReliability {
             let redis_client = redis::Client::open(dsn.clone()).map_err(|e| {
                 ApcErrorKind::GeneralError(format!("Could not connect to redis server: {:?}", e))
             })?;
-            Some(Arc::new(redis_client))
+            //Verify that we can connect to the server.
+            match redis_client.get_connection_with_timeout(CONNECTION_EXPIRATION) {
+                Ok(_) => {
+                    info!("🔍 Connected to reliability server at {dsn}");
```

Comment:
Could the `dsn` have a username and password included? If so, we might want to scrape this from the logs.

Reply:
No, the DSN should never have the account info in it. (Of course, for anyone doing this stand-alone, then there's the potential, but then they shouldn't be running logging at `info` either.